MaxText/configs/base.yml

```diff
@@ -44,20 +44,20 @@ int8_training: False
 # Global parameter scale needs to be a power of 2. If you want finer grained control of the model sizes 
 # then you should explicitly set base_embed_dim, base_num_heads, base_mlp_dim, base_num_decoder_layers and/or head_dim.
 global_parameter_scale: 1
-base_emb_dim: 2560
```

Comment:
Just to make sure we're on the same page, are you working with Mohit to add forking logic where if the model is set to say "llama2-7b" these overrides go into effect? The overrides can be hard coded into pyconfig.

Reply:
utilizing Mohit's prototype